src/components/ShareList.jsx

```diff
@@ -0,0 +1,58 @@
+import { useState } from 'react';
+import { shareList, useAuth } from '../api';
+
+export function ShareList() {
+	const [emailData, setEmailData] = useState('');
```

Comment:
Same as [this](https://github.com/the-collab-lab/tcl-75-smart-shopping-list/pull/38/files#r1747611186)!